r/.styler_excludes.R

```diff
@@ -15,4 +15,4 @@
 #  specific language governing permissions and limitations
 #  under the License.
 
-c("tests/testthat/latin1.R", "data-raw/codegen.R")
\ No newline at end of file
+c("data-raw/codegen.R")
```

Comment:
Can we remove the `c()` now there's only one thing here?

Reply:
Sure, I left it like this to communicate how to add other files (as vector vs line by line similar to .ignore-files) but it's an R file so maybe that is clear enough.